org.eclipse.xtext.ide.tests/testlang-src-gen/org/eclipse/xtext/ide/tests/testlanguage/serializer/PartialSerializationTestLanguageSyntacticSequencer.java

```diff
@@ -71,17 +71,15 @@ else if (match_OptionalValue_Kw2Keyword_3_q.equals(syntax))
 	 * This ambiguous syntax occurs at:
 	 *     (rule start) '#1' (ambiguity) '{' 'ref' ref=[Node|QualifiedName]
 	 *     (rule start) '#1' (ambiguity) '{' children+=Node
-	 *     (rule start) '#1' (ambiguity) (';' | ('{' '}')) (rule start)
```

Comment:
is this caused by the fix for stable order?

Reply:
I'm optimistic :)